templates/quickstart-hashicorp-nomad-cluster.template.yaml

```diff
@@ -0,0 +1,666 @@
+      Parameters:
+        PrivateSubnet1ID: !Ref 'PrivateSubnet1ID'
+        PrivateSubnet2ID: !Ref 'PrivateSubnet2ID'
+        PrivateSubnet3ID: !Ref 'PrivateSubnet3ID'
+        PublicSubnet1ID: !Ref 'PublicSubnet1ID'
+        PublicSubnet2ID: !Ref 'PublicSubnet2ID'
+        PublicSubnet3ID: !Ref 'PublicSubnet3ID'
+        VPCID: !Ref 'VPCID'
+        NomadServerInstanceType: !Ref 'NomadServerInstanceType'
+        NomadServerNodeCount: !Ref 'NomadServerNodeCount'
+        NomadServerSecurityGroup: !Ref 'NomadSecGroup'
+        KeyPairName: !Ref 'KeyPairName'
+        ConsulEc2RetryTagKey: !If [DeployConsul, !Ref 'ConsulEc2RetryTagKey', '']
+        ConsulEc2RetryTagValue: !If [DeployConsul, !Ref 'ConsulEc2RetryTagValue', '']
+        ConsulDatacenter: !If [DeployConsul, !Ref 'ConsulDatacenter', '']
+        EnableConsulServiceMesh: !If [DeployConsul, !Ref 'EnableConsulServiceMesh', 'false']
+        NomadEc2RetryTagKey: !Ref 'NomadEc2RetryTagKey'
+        NomadEc2RetryTagValue: !Ref 'NomadEc2RetryTagValue'
+        NomadDatacenter: !Ref 'NomadDatacenter'
+        QSS3BucketName: !Ref 'QSS3BucketName'
+        QSS3KeyPrefix: !Ref 'QSS3KeyPrefix'
+        SSLCertificateId: !If
+              - GenerateSSL
+              - !GetAtt 'ConfigureSSLStack.Outputs.ACMCertificate'
+              - !Ref 'SSLCertificateArn'
+
+  NomadClients:
+    Type: 'AWS::CloudFormation::Stack'
+    DependsOn: NomadServers
+    Properties:
+      TemplateURL: !Sub >-
+        https://${QSS3BucketName}.s3.amazonaws.com/${QSS3KeyPrefix}templates/quickstart-hashicorp-nomad-clients.template.yaml
+      Parameters:
+        PrivateSubnet1ID: !Ref 'PrivateSubnet1ID'
+        PrivateSubnet2ID: !Ref 'PrivateSubnet2ID'
+        PrivateSubnet3ID: !Ref 'PrivateSubnet3ID'
+        NomadClientInstanceType: !Ref 'NomadClientInstanceType'
+        NomadClientNodeCount: !Ref 'NomadClientNodeCount'
+        NomadClientSecurityGroup: !Ref 'NomadSecGroup'
+        ConsulEc2RetryTagKey: !If [DeployConsul, !Ref 'ConsulEc2RetryTagKey', '']
+        ConsulEc2RetryTagValue: !If [DeployConsul, !Ref 'ConsulEc2RetryTagValue', '']
+        EnableConsulServiceMesh: !If [DeployConsul, !Ref 'EnableConsulServiceMesh', 'false']
+        ConsulDatacenter: !If [DeployConsul, !Ref 'ConsulDatacenter', '']
+        NomadDatacenter: !Ref 'NomadDatacenter'
+        EnableRawExec: !Ref 'EnableRawExecDriver'
+        KeyPairName: !Ref 'KeyPairName'
+        NomadEc2RetryTagKey: !Ref 'NomadEc2RetryTagKey'
+        NomadEc2RetryTagValue: !Ref 'NomadEc2RetryTagValue'
+        QSS3BucketName: !Ref 'QSS3BucketName'
+        QSS3KeyPrefix: !Ref 'QSS3KeyPrefix'
+
+  LoadBalancerFQDNRecord:
+    Condition: SetupRoute53
+    Type: AWS::Route53::RecordSet
+    Properties:
+      Type: A
+      Name: !Ref 'LoadBalancerFQDN'
+      AliasTarget:
+        HostedZoneId: !GetAtt 'NomadServers.Outputs.ALBCanonicalHostedZoneNameID'
```

Comment:
output name not available, should be `NomadALBCanonicalHostedZoneID` and `NomadALBDNSName`